lib/clustering/mean_shift.h

```diff
@@ -9,19 +9,39 @@
 #include "clustering.h"
 #include <vector>
 #include "generic/cluster.h"
+#include <Eigen/Core>
 
 namespace MouseTrack {
 
 class MeanShift: public Clustering {
 public:
 
   /// k is the number of desired clusters
-  MeanShift(const int k);
+  MeanShift(double window_size);
   /// Splits a point cloud into k clusters randomly
   std::vector<Cluster> operator()(const PointCloud& cloud) const;
 
 private:
-    const int _k;
+  // when two peaks are closer than this, they are merged.
```

Comment:
tripple slashes "///" if doxygen should show documentation

Reply:
Do we want to show documentation for private methods in the Doxygen as well?

Reply:
oh, so that was consciously, I thought you forgot it. I don't mind hiding it. I usually make doc comments available to doxygen (I think it has its own flags where it can hide private stuff if desired).

Reply:
For me it doesn't matter, but I think we should be consistent. Probably the easiest way is to show everything and then hide the stuff we don't want afterwards.